src/nodejs/managed-identity/azure-vm/README.md

```diff
@@ -1,6 +1,6 @@
 # Managed Identity for Azure VM Sample
 
-This sample demonstrates how to use [managed identity via the msal-node library](/lib/msal-node/docs/managed-identity.md) to retrieve tokens for a managed identity application running on an Azure VM.
+This sample demonstrates how to use [managed identity via the msal-node library](/lib/msal-node/docs/managed-identity.md) to acquire a token for accessing Azure KeyVault, on behalf of a managed identity configured on an Azure VM. The sample then calls the downstream api - it calls KeyVault and retrieves a secret.
```

Comment:
```suggestion
This sample demonstrates how to use [managed identity via the msal-node library](/lib/msal-node/docs/managed-identity.md) to acquire a token for accessing Azure Key Vault, on behalf of a managed identity configured on an Azure virtual machine (VM). The sample then calls the downstream API, which calls Azure Key Vault and retrieves a secret.
```

Reply:
@Robbie-Microsoft - also, the link to the managed identity document is broken. Looking at the folder (https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-node/docs) doesn't seem like there is a doc for managed identity?

Reply:
Shoot, it looks like my most recent updates did not make it to this PR. I'll try-reuploading.

Reply:
There is, but the doc is part of the msal-node PR that merges the MSI functionality into the dev branch. The link will work when the PR is committed. Otherwise, there's no way to view the file in the meantime.

Reply:
@Robbie-Microsoft do we have a ETA for the file being live?

Reply:
Well... if GA is scheduled for the end of March, then that's the ETA. The only thing holding this back right now is manual testing of Service Fabric.

Reply:
Let's remove the link until the page is available. We can upate the sample later.